### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by nhat15**
init project

### CODE REVIEW

**Comment by nhat15** [comment]
ready

**Comment by nhat15** [comment]
ready

**Comment by nhat15** [comment]
ready

**Comment by nhat15** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (huongnt-2545) [inline app/views/layouts/application.html.erb:9]: dùng nháy kép

File: config/database copy.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# MySQL. Versions 5.5.8 and up are supported.
```
Comment (huongnt-2545) [inline config/database copy.yml:1]: tên file -> `databse.yml.example`

**Review by huongnt-2545 — COMMENTED** [review]
commented

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**